app/controllers/index.rb

```diff
@@ -1,4 +1,11 @@
 get '/' do
```

Comment:
this home route should probably not do anything, but simply redirect to `/missions` right away so that we can control what's on our home page and still be RESTful